pkg/utils/pod/pod_utils_test.go

```diff
@@ -96,3 +101,93 @@ func TestContainerRestarted(t *testing.T) {
 		})
 	}
 }
+
+func MakePod(setName, groupIndex, workerIndex, namespace string) *corev1.Pod {
```

Comment:
This introduces an import cycle since the integration tests import the pod/accelerator utils for the env var name consts. Also, utils/accelerators/tpu_test.go similarly redefines many pod spec util functions.

Moving the env var consts to their own package under utils should allow us to refactor some of that test code under test/testutils. I can refactor the test code in pod/accelerator utils in a follow up PR.

Reply:
Make a lot of sense.

Reply:
Based on Abdullah's suggestion I cleaned up the import cycle and moved this func.